tests/test_swift.py

```diff
@@ -53,7 +53,7 @@ def test_parser(tmpdir):
     # Only run this test if swiftc is on the path.
     if shutil.which("swiftc"):
         for filepath in tmpdir.glob("*.swift"):
-            subprocess.check_call(["swiftc", "-dump-parse", filepath])
+            subprocess.check_call(["swiftc", "-parse", filepath])
```

Comment:
What's the context on this change?  Also, I don't think this runs in CI (which is currently Linux-only), and maybe we should address that at some point.

Reply:
I _think_ that locally my Swift updated and thus causing test failures when it uses `-dump-parse` (as it expects `GleanMetrics` and other symbols to be present). `-parse` on the other hand really just parses, without looking up symbols.

We once did think about getting that test on CI, but

* getting swift on Linux is ... not easy
* running on macOS is more expensive